apps/web/src/components/Common/CategoryFilters.tsx

```diff
@@ -38,14 +38,31 @@ const CategoryFilters = () => {
 
   const slide = (shift: number) => {
     Analytics.track(TRACK.CLICK_CATEGORIES_SCROLL_BUTTON)
+
     if (scrollRef.current) {
+      /**
+       * 
+       * For reference : https://stackoverflow.com/questions/65016565/react-why-is-that-changing-the-current-value-of-ref-from-useref-doesnt-trigger
+       * 
+      I observed that since updating the Ref doesn't re-render the UI,
+      the scrollX/ChevronLeftOutline was still invisible after the FIRST CLICK on scrollEnd/ChevronRightOutline
+      Also, this calculation:
+              scrollRef.current.scrollLeft += shift
+      did not update the value of scrollRef.current.scrollLeft on the go
+       */
+
+      // That's why I used a new variable to handle the calculation of "shift" effectively
+      let scrolled = scrollRef.current.scrollLeft + shift
```

Comment:
can be `const` if we are not reassigning the value 🙇🏻

Reply:
Sure! Let me update that, I overlooked the type after the whole debugging I guess.